### PR TITLE
MoneyHelper theming

### PR DESCRIPTION
**Pull request by AnthonyG4**

### CODE REVIEW

**Comment by masbuild** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">There is no Targetprocess ticket associated to this PR. Please include the link to the corresponding task (TP-XXXX). If no ticket matches these changes consider creating one and linking it here.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is huge. Consider next time to split up the workinto multiple PRs. A good guide about small PRs can be seenhere(link to mas-standards - small PRs guide)</td>
    </tr>
  </tbody>
</table>

At Money Advice Service, we use Danger gem, to ensure that our commits and pull requests follow [standards](https://github.com/moneyadviceservice/mas-standards/blob/master/Dangerfile). If you think one of the points here has been raised in error (or shouldn't be in our standards), open an issue or pull request on [our Danger config](https://github.com/moneyadviceservice/mas-standards).

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: config/locales/land_and_buildings_transaction_tax.cy.yml [hunk 1]
```diff
@@ -3,15 +3,15 @@ cy:
     meta:
       title: "Trafodiadau Tir ac Adeiladau (LBTT) – Cyfrifiannell Treth Stamp yr Alban"
       description: "Defnyddiwch ein cyfrifiannell Treth Trafodiadau Tir ac Adeiladau (LBTT) i gyfrifo faint o LBTT (Treth Stamp yn flaenorol) y bydd angen ichi ei thalu ar eich cartref yn yr Alban"
-      canonical_url: "https://www.moneyadviceservice.org.uk/cy/tools/prynu-ty/cyfrifiannell-treth-trafodion-tir-ac-adeiladau-alban"
+      canonical_url: "https://www.moneyhelper.org.uk/cy/tools/prynu-ty/cyfrifiannell-treth-trafodion-tir-ac-adeiladau-alban"
```
Comment (benlovell) [inline config/locales/land_and_buildings_transaction_tax.cy.yml:6]: @Venetia-maps what should this be?

File: config/locales/land_and_buildings_transaction_tax.en.yml [hunk 2]
```diff
@@ -3,15 +3,15 @@ en:
     meta:
       title: "Land and Buildings Transaction Tax (LBTT) – Scottish Stamp Duty calculator"
       description: "Use our Land and Buildings Transaction Tax (LBTT) calculator to calculate how much LBTT (formerly Stamp Duty) you’ll need to pay on your home in Scotland"
-      canonical_url: "https://www.moneyadviceservice.org.uk/en/tools/house-buying/land-and-buildings-transaction-tax-calculator-scotland"
+      canonical_url: "https://www.moneyhelper.org.uk/en/tools/house-buying/land-and-buildings-transaction-tax-calculator-scotland"
```
Comment (benlovell) [inline config/locales/land_and_buildings_transaction_tax.en.yml:6]: @Venetia-maps ?

File: config/locales/land_transaction_tax.cy.yml [hunk 3]
```diff
@@ -3,7 +3,7 @@ cy:
     meta:
       title: "Treth Trafodiadau Tir (LTT) - Cyfrifiannell Treth Stamp Cymru"
       description: "Defnyddiwch ein cyfrifiannell Treth Trafodiadau Tir (LTT) i gyfrifo faint o LTT (Treth Stamp yn flaenorol) y bydd angen ichi ei thalu ar eich cartref yn yr Alban"
-      canonical_url: "https://www.moneyadviceservice.org.uk/cy/tools/prynu-ty/cyfrifiannell-treth-trafodiadau-tir-cymru"
+      canonical_url: "https://www.moneyhelper.org.uk/cy/tools/prynu-ty/cyfrifiannell-treth-trafodiadau-tir-cymru"
```
Comment (benlovell) [inline config/locales/land_transaction_tax.cy.yml:6]: @Venetia-maps

File: config/locales/land_transaction_tax.en.yml [hunk 4]
```diff
@@ -3,16 +3,16 @@ en:
     meta:
       title: "Land Transaction Tax (LTT) - Welsh Stamp Duty Calculator"
       description: "Use our Land Transaction Tax (LTT) calculator to calculate how much LTT (formerly Stamp Duty) you'll need to pay on your home in Scotland"
-      canonical_url: "https://www.moneyadviceservice.org.uk/en/tools/house-buying/land-transaction-tax-calculator-wales"
+      canonical_url: "https://www.moneyhelper.org.uk/en/tools/house-buying/land-transaction-tax-calculator-wales"
```
Comment (benlovell) [inline config/locales/land_transaction_tax.en.yml:6]: @Venetia-maps

File: config/locales/mortgage_calculator.cy.yml [hunk 5]
```diff
@@ -3,7 +3,7 @@ cy:
     meta:
       title: "Cyfrifiannell Morgais - Cyfrifwch eich ad-daliadau morgais a'ch taliadau llog"
       description: "Defnyddiwch ein Cyfrifiannell Morgais i ganfod faint fydd eich taliadau morgais misol yn cynnwys yr ad-daliad a'r taliad llog."
-      canonical_url: "https://www.moneyadviceservice.org.uk/cy/tools/cyfrifiannell-morgais"
+      canonical_url: "https://www.moneyhelper.org.uk/cy/homes/buying-a-home/mortgage-calculator"
```
Comment (benlovell) [inline config/locales/mortgage_calculator.cy.yml:6]: @Venetia-maps this looks ok?

File: config/locales/stamp_duty.cy.yml [hunk 6]
```diff
@@ -3,7 +3,7 @@ cy:
     meta:
       title: "Cyfrifiannell Treth Stamp - Cyfrifwch y cyfraddau newydd a ddiweddarwyd ar gyfer Treth Dir y Doll Stamp"
       description: "Defnyddiwch ein cyfrifiannell Treth Stamp i gael amcangyfrif o faint o Dreth Dir y Doll Stamp y bydd angen i chi ei thalu ar eich cartref newydd yn seiliedig ar y cyfraddau newydd a ddiweddarwyd"
-      canonical_url: "https://www.moneyadviceservice.org.uk/cy/tools/prynu-ty/cyfrifiannell-treth-stamp"
+      canonical_url: "https://www.moneyhelper.org.uk/cy/homes/buying-a-home/stamp-duty-calculator"
```
Comment (benlovell) [inline config/locales/stamp_duty.cy.yml:6]: @Venetia-maps seems ok?

File: app/helpers/mortgage_calculator/application_helper.rb [hunk 7]
```diff
@@ -12,25 +12,25 @@ def calculator_name
 
     def full_mortgage_calculator_url
       if I18n.locale == :cy
-        "https://www.moneyadviceservice.org.uk/cy/tools/cyfrifiannell-morgais"
+        "https://www.moneyhelper.org.uk/cy/homes/buying-a-home/mortgage-calculator"
```
Comment (Venetia-maps) [inline app/helpers/mortgage_calculator/application_helper.rb:15]: no, this was some kind of weird blanket thing that they did to change moneyadviceservice to moneyhelper but they are completely wrong

**Review by Venetia-maps — COMMENTED** [review]
The canonicals had some kind of weird blanket thing that they did to change moneyadviceservice to moneyhelper but they are completely wrong

File: config/locales/land_transaction_tax.en.yml (hunk 4 above)
Comment (Venetia-maps) [reply config/locales/land_transaction_tax.en.yml:6]: this is not right should be  https://www.moneyhelper.org.uk/en/homes/buying-a-home/land-transaction-tax-calculator-wales

File: config/locales/mortgage_calculator.cy.yml (hunk 5 above)
Comment (Venetia-maps) [reply config/locales/mortgage_calculator.cy.yml:6]: yes - ok

File: config/locales/stamp_duty.cy.yml (hunk 6 above)
Comment (Venetia-maps) [reply config/locales/stamp_duty.cy.yml:6]: OK

File: config/locales/land_transaction_tax.cy.yml (hunk 3 above)
Comment (Venetia-maps) [reply config/locales/land_transaction_tax.cy.yml:6]: https://www.moneyhelper.org.uk/cy/homes/buying-a-home/land-transaction-tax-calculator-wales

File: config/locales/land_and_buildings_transaction_tax.cy.yml (hunk 1 above)
Comment (Venetia-maps) [reply config/locales/land_and_buildings_transaction_tax.cy.yml:6]: https://www.moneyhelper.org.uk/cy/homes/buying-a-home/land-and-buildings-transaction-tax-calculator-scotland

File: config/locales/stamp_duty.cy.yml [hunk 8]
```diff
@@ -91,22 +91,22 @@ cy:
     next_steps:
       learn_more:
         title: "A wyddech chi?"
-        tip_1: "Rhaid i chi dalu Treth Stamp cyn pen 14 diwrnod ar ol prynu eiddo. Os ydych yn defnyddio cyfreithiwr i gwblhau'r trawsgludo, bydd fel arfer yn trefnu'r taliad ar eich rhan"
+        tip_1_html: "Rhaid i chi dalu Treth Stamp cyn pen 14 diwrnod ar ol prynu eiddo. Os ydych yn defnyddio cyfreithiwr i gwblhau'r trawsgludo, bydd fel arfer yn trefnu'r taliad ar eich rhan"
         url_online_portal: https://portal.revenue.scot/rslive/en/login
```
Comment (Venetia-maps) [inline config/locales/stamp_duty.cy.yml:95]: links need to open in a new window

**Review by Venetia-maps — COMMENTED** [review]
havent finished yet

File: config/locales/affordability.en.yml [hunk 9]
```diff
@@ -2,8 +2,8 @@ en:
   affordability:
     meta:
       title: Mortgage affordability calculator – How much mortgage can you afford to borrow?
-      description: Use the Money Advice Service mortgage affordability calculator to find out how much mortgage you can afford to borrow for your new house
-      canonical_url: "https://www.moneyadviceservice.org.uk/en/tools/house-buying/mortgage-affordability-calculator"
+      description: Use the Money Helper mortgage affordability calculator to find out how much mortgage you can afford to borrow for your new house
```
[the remark this answers is not in the document]
Comment (Venetia-maps) [reply config/locales/affordability.en.yml:5]: Canonical is wrong 
https://www.moneyhelper.org.uk/en/homes/buying-a-home/mortgage-affordability-calculator

File: config/locales/affordability.en.yml [hunk 10]
```diff
@@ -173,88 +173,89 @@ en:
...
           heading: "What lenders look for when you apply"
           info_html: "How much can you really afford to borrow?"
-          url: "https://www.moneyadviceservice.org.uk/en/articles/how-much-can-you-afford-to-borrow"
+          url: "https://www.moneyhelper.org.uk/en/articles/how-much-can-you-afford-to-borrow"
         tip_2:
           heading: "Make changes to your spending"
           info_html: "Use our %{budget_planner} to get the full picture"
           url_text: "Budget Planner"
-          url: "https://www.moneyadviceservice.org.uk/en/tools/budget-planner"
+          url: "https://www.moneyhelper.org.uk/en/tools/budget-planner"
```
[the remark this answers is not in the document]
Comment (Venetia-maps) [reply config/locales/affordability.en.yml:230]: Needs to open in a new window

**Pull request merged**